src/theme.js

```diff
@@ -1,243 +1,252 @@
+      "notifications.border"               : color.border.default,
+      "notificationsErrorIcon.foreground"  : color.danger.fg,
+      "notificationsWarningIcon.foreground": color.attention.fg,
+      "notificationsInfoIcon.foreground"   : color.accent.fg,
+
+      "pickerGroup.border"    : color.border.default,
+      "pickerGroup.foreground": color.fg.muted,
+      "quickInput.background" : color.canvas.overlay,
+      "quickInput.foreground" : color.fg.default,
+
+      "statusBar.foreground"             : color.fg.muted,
+      "statusBar.background"             : color.canvas.default,
+      "statusBar.border"                 : color.border.default,
+      "statusBar.noFolderBackground"     : color.canvas.default,
+      "statusBar.debuggingBackground"    : color.danger.emphasis,
+      "statusBar.debuggingForeground"    : color.fg.onEmphasis,
+      "statusBarItem.prominentBackground": color.canvas.subtle,
+
+      "editorGroupHeader.tabsBackground": color.canvas.inset,
+      "editorGroupHeader.tabsBorder"    : color.border.default,
+      "editorGroup.border"              : color.border.default,
+
+      "tab.activeForeground"        : color.fg.default,
+      "tab.inactiveForeground"      : color.fg.muted,
+      "tab.inactiveBackground"      : color.canvas.inset,
+      "tab.activeBackground"        : color.canvas.default,
+      "tab.hoverBackground"         : color.canvas.default,
+      "tab.unfocusedHoverBackground": color.neutral.subtle,
+      "tab.border"                  : color.border.default,
+      "tab.unfocusedActiveBorderTop": color.border.default,
+      "tab.activeBorder"            : color.canvas.default,
+      "tab.unfocusedActiveBorder"   : color.canvas.default,
+      "tab.activeBorderTop"         : color.primer.border.active,
+
+      "breadcrumb.foreground"               : color.fg.muted,
+      "breadcrumb.focusForeground"          : color.fg.default,
+      "breadcrumb.activeSelectionForeground": color.fg.muted,
+      "breadcrumbPicker.background"         : color.canvas.overlay,
+
+      "editor.foreground"                 : color.fg.default,
+      "editor.background"                 : color.canvas.default,
+      "editorWidget.background"           : color.canvas.overlay,
+      "editor.foldBackground"             : color.neutral.muted,
+      "editor.lineHighlightBackground"    : color.codemirror.activelineBg,
+      "editor.lineHighlightBorder"        : onlyDarkHighContrast(color.accent.fg),
+      "editorLineNumber.foreground"       : color.codemirror.linenumberText,
+      "editorLineNumber.activeForeground" : color.fg.default,
+      "editorIndentGuide.background"      : color.border.muted,
+      "editorIndentGuide.activeBackground": color.border.default,
+      "editorWhitespace.foreground"       : color.fg.subtle,
+      "editorCursor.foreground"           : color.accent.fg,
+
+      "editor.findMatchBackground"            : color.attention.muted,
```

Comment:
Same as above, I wonder if it's safer to use something like `alpha(color.attention.emphasis, 0.3)` to make sure they always stay semi-transparent even if Primitives changes. That said, I don't expect these global variables to change much, so maybe fine? 🤔

Reply:
I've reverted those back to alphas.